samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -50,6 +50,8 @@ export interface HomeScreenProps {
     outboundParticipants?: string[];
     /* @conditional-compile-remove(PSTN-calls) */
     alternateCallerId?: string;
+    /* @conditional-compile-remove(teams-identity-support) */
+    teamsToken?: string;
```

Comment:
nit: possibly rename to teamsIdentityToken?

Reply:
I checked the word they use in teams scenario, they still call it token or access token just like ACS, so I guess add a prefix "Teams" would align with their naming

For now if you wanna get teams token, there are 2 ways, 

1. go to azure portal then generate one:
![image](https://user-images.githubusercontent.com/11863655/208215072-21e5a48e-1c50-4986-9263-c57075b8c025.png)

2. Set up a service following the instruction from https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/manage-teams-identity?pivots=programming-language-javascript, then you would get your teams token when you log into this service using OAuth or some other ways